templates/components/footer.html.ep

```diff
@@ -0,0 +1,17 @@
+ <footer class="main-footer">
+  <small>
```

Comment:
Add comment with "copy right notice" and "privacy links and version" for the other section